guides/common/modules/proc_importing-a-subscription-manifest-into-foreman-server.adoc

```diff
@@ -7,12 +7,6 @@ ifdef::foreman-el,katello[]
 This is for users of the Katello plug-in and Red Hat operating systems only.
 endif::[]
 
-[NOTE]
-====
-Simple Content Access (SCA) is set on the organization, not the manifest.
-Importing a manifest does not change your organization's Simple Content Access status.
-====
-
```

Comment:
I am not sure if we should remove this because SCA will always be enabled. Here we are talking about its impact on manifest.

Reply:
I would say still remove this. The reason for the note originally was so users wouldn't be confused that toggling SCA locally didn't change their manifest on access.redhat.com. Now that's not a concern.